pkg/common/watchdog_test.go

```diff
@@ -0,0 +1,22 @@
+package common_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/matrix-org/waterfall/pkg/common"
+)
+
+func BenchmarkWatchdogChannel_Notify(b *testing.B) {
+	watchdogConfig := common.WatchdogConfig{
+		Timeout:   2 * time.Second,
+		OnTimeout: func() {},
+	}
+	w := common.StartWatchdog(watchdogConfig)
+
+	// run the Fib function b.N times
```

Comment:
Sorry, what is the `Fib` function?

Reply:
Should mean Notify Function. I'll change this.